kafka-connect-mysql-v60/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG KAFKA_CONNECT_VERSION=6.0.2
+
+FROM confluentinc/cp-kafka-connect-base:${KAFKA_CONNECT_VERSION}
+
+ARG KAFKA_CONNECT_VERSION=6.0.2
+ARG MYSQL_CONNECTOR_VERSION=8.0.25
+
+LABEL version="${KAFKA_CONNECT_VERSION}-${MYSQL_CONNECTOR_VERSION}"
+LABEL maintainer="ozaki@chatwork.com"
+LABEL maintainer="murakami@chatwork.com"
+
+USER root
+
+RUN confluent-hub install --no-prompt confluentinc/kafka-connect-jdbc:10.2.0 \
```

Comment:
https://docs.confluent.io/platform/current/installation/docker/development.html#create-a-docker-image-containing-c-hub-connectors
`confluent-hub` is now installed from v6.x.